docs/setup/turn/eturnal.md

```diff
@@ -0,0 +1,145 @@
+# eturnal TURN server
+
+The following sections describe how to install [eturnal](<https://github.com/processone/eturnal>) (which implements the TURN REST API).
+
+## `eturnal` setup
+
+### Initial installation
+
+The `eturnal` TURN server implementation is available from a variety of sources such as native package managers, binary packages, installation from source or [container image](https://eturnal.net/documentation/code/docker.html).
+
+On **APT-based** Linux distributions, run:
+
+    sudo apt install extrepo
+    sudo extrepo enable eturnal
+    sudo apt update
+    sudo apt install eturnal
```

Comment:
````suggestion
```sh
sudo apt install extrepo
sudo extrepo enable eturnal
sudo apt update
sudo apt install eturnal
```
````

Reply:
There are more of this kind of changes in this file.